client/src/components/App/Content/InvestigationForm/TabManagement/ClinicalDetails/ClinicalDetails.tsx

```diff
@@ -76,8 +78,14 @@ const ClinicalDetails: React.FC<Props> = ({ id,isViewMode }: Props): JSX.Element
 
     const saveForm = (e: any) => {
         e.preventDefault();
-        if (clinicalDetails)
-         saveClinicalDetailsAndDeleteContactEvents(clinicalDetails, id);
+        if (clinicalDetails && !isViewMode){
+            saveClinicalDetailsAndDeleteContactEvents(clinicalDetails, id);
+        }
+        else{
```

Comment:
You should add "if isViewMode" in the else statement

Reply:
right, added.